packages/api/src/domain/medical/facility.ts

```diff
@@ -28,6 +28,11 @@ export interface FacilityCreate extends Omit<BaseDomainCreate, "id"> {
   data: FacilityData;
 }
 
+export interface FacilityRegister extends FacilityCreate {
+  id?: string;
+  cwFacilityName?: string;
```

Comment:
I honestly don't remember why do we have this. Maybe you can add a TSDoc to this prop if you remember why?

Reply:
I dont remember either but based on the logic it seems like its just an option to name the facility rather than using `facility.data.name`

Reply:
nit: the TSDoc will be more useful (visible when we hover over it) if we add it to the prop, no the type/interface.